### DISCUSSION TITLE
v7.5.0-preview.2 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#21266] [General]
### Engine Updates and Fixes

- Fix `using assembly` to use `Path.Combine` when constructing assembly paths (#21169)
- Validate the value for `using namespace` during semantic checks to prevent declaring invalid namespaces (#21162)

### General Cmdlet Updates and Fixes

- Add `WinGetCommandNotFound` and `CompletionPredictor` modules to track usage (#21040)
- `ConvertFrom-Json`: Add `-DateKind` parameter (#20925) (Thanks @jborean93!)
- Add tilde expansion for windows native executables (#20402) (Thanks @domsleee!)
- Add `DirectoryInfo` to the `OutputType` for `New-Item` (#21126) (Thanks @MartinGC94!)
- Fix `Get-Error` serialization of array values (#21085) (Thanks @jborean93!)

### Code Cleanup

<details>

<summary>

<p>We thank the following contributors!</p>
<p>@eltociear</p>

</summary>

<ul>
<li>Fix a typo in <code>CoreAdapter.cs</code> (#21179) (Thanks @eltociear!)</li>
<li>Remove <code>PSScheduledJob</code> module source code (#21189)</li>
</ul>

</details>

### Tests

- Rewrite the mac syslog tests to make them less flaky (#21174)

### Build and Packaging Improvements

<details>

<summary>
<p>Bump to .NET 9 Preview 1</p>
<p>We thank the following contributors!</p>
<p>@gregsdennis</p>

</summary>

<ul>
<li>Bump to .NET 9 Preview 1 (#21229)</li>
<li>Add dotnet-runtime-9.0 as a dependency for the Mariner package</li>
<li>Add dotenv install as latest version does not work with current Ruby version (#21239)</li>
<li>Remove <code>surrogateFile</code> setting of APIScan (#21238)</li>
<li>Update experimental-feature json files (#21213)</li>
<li>Update to the latest NOTICES file (#21236)(#21177)</li>
<li>Update the cgmanifest (#21237)(#21093)</li>
<li>Update the cgmanifest (#21178)</li>
<li>Bump XunitXml.TestLogger from 3.1.17 to 3.1.20 (#21207)</li>
<li>Update versions of PSResourceGet (#21190)</li>
<li>Generate MSI for <code>win-arm64</code> installer (#20516)</li>
<li>Bump JsonSchema.Net to v5.5.1 (#21120) (Thanks @gregsdennis!)</li>
</ul>

</details>

### Documentation and Help Content

- Update `README.md` and `metadata.json` for v7.5.0-preview.1 release (#21094)
- Fix incorrect examples in XML docs in `PowerShell.cs` (#21173)
- Update WG members (#21091)
- Update changelog for v7.4.1 (#21098)

[7.5.0-preview.2]: https://github.com/PowerShell/PowerShell/compare/v7.5.0-preview.1...v7.5.0-preview.2

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - EB8731E878565EDD22246A12C1D7174EB34A52A50A0D5DA1C91A6C31411060CE
- powershell-7.5.0-preview.2-linux-arm32.tar.gz
  - BF19F8AD881D3CA7323BBB71E65F8148DE21E109E57D9C90510FAE8D5627EB4E
- powershell-7.5.0-preview.2-linux-arm64.tar.gz
  - A52E0179CA5DCA2FB2117037AFC8ABF2718AB9A6440309CDA5FAD2113AFA510A
- powershell-7.5.0-preview.2-linux-musl-x64.tar.gz
  - 370C774AF99D46CB1B36DE6DE8DB3079DA2244ECFE544524D7A5BBBD7D4D1A28
- powershell-7.5.0-preview.2-linux-x64-fxdependent.tar.gz
  - 2CB85B4D4D58F5B6E76B2EAEEFDB6F95D57F207A4CD9E49EB159AD5144CFDC46
- powershell-7.5.0-preview.2-linux-x64-musl-noopt-fxdependent.tar.gz
  - 3DA5F08C917029DF9584CB3F4B8DF6C07ABA27B8878C4DDEEF76CC4C5479360E
- powershell-7.5.0-preview.2-linux-x64.tar.gz
  - 7377D3539AE8D52897E3EF6C8B5C44AD463E708F90E9888D2E057B96B96B6D33
- powershell-7.5.0-preview.2-osx-arm64.pkg
  - A3FB387D86FACC328B0639D9636B96795DB88AB3D543662DB8FD4AB26424D12D
- powershell-7.5.0-preview.2-osx-arm64.tar.gz
  - 1D2C7387F61650E04B1A42E03857ABB730284CF78F93D543CAA082AD8D5B5EF1
- powershell-7.5.0-preview.2-osx-x64.pkg
  - 2217CEC96708C17B924987EE9625296E212FF2CF4539AB0EB94D6268DDE65B91
- powershell-7.5.0-preview.2-osx-x64.tar.gz
  - 2AEEE26FFB173C4015984C950A1C29112964D18E7B5B3EBCCF7F5598B176A618
- PowerShell-7.5.0-preview.2-win-arm64.msi
  - 6DD8C63C41F202C6A5EE62050DC7930392DDE38D8203F32D517DC12DC6357FEB
- PowerShell-7.5.0-preview.2-win-arm64.zip
  - 54D282F5AF2BE2B5F596829D7232FAACA7913849F63D6CD409DDD8D7AE58A06D
- PowerShell-7.5.0-preview.2-win-fxdependent.zip
  - E0CDF33B710C7BDDFE9EF1C9054FB21E2022FACAF8BE20FE32080D0DD79FE11D
- PowerShell-7.5.0-preview.2-win-fxdependentWinDesktop.zip
  - 6E65FEBB91256DCEE5F9B7EF82B0E782DBC4A56B0263256D1EAA5E64CD2B977C
- PowerShell-7.5.0-preview.2-win-x64.msi
  - A107F34D9D6E41259D13C891CD2C4062BD5C0FB103A00AD1E525A503B9060897
- PowerShell-7.5.0-preview.2-win-x64.zip
  - E3B3025982632B9C8B1BAC4AB16C72623E41E4DB6D2CB770C6E5D22304CDD5F4
- PowerShell-7.5.0-preview.2-win-x86.msi
  - C2DFF5BFD7991721B0F40415B5F275AC9D42540A35ECA8489A78659D87FAADEB
- PowerShell-7.5.0-preview.2-win-x86.zip
  - E322270099ED943019A009C81AB6827F3381A5219178C4EF3793975F90A87E1D
- powershell-preview_7.5.0-preview.2-1.deb_amd64.deb
  - 8925C4A36F1EA1D5FF85609FB131A61227CA9767DFFC8E683B4CAB3773317F61
- powershell-preview-7.5.0_preview.2-1.cm.aarch64.rpm
  - EF78BD4ACDED12AF007101288054E5E98E2DE77DEE23E3104DB3DD04235661C2
- powershell-preview-7.5.0_preview.2-1.cm.x86_64.rpm
  - C90024CAC8311CA85463B17E1DB326464DA9CCBC8D48C73B67BAFDA6C23586DC
- powershell-preview-7.5.0_preview.2-1.rh.x86_64.rpm
  - 735FE4014EA6A46088A26D050276DE009AD62A2CD9362944D31177D2817C5AB5

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.5.0-preview.2'>v7.5.0-preview.2 Release of PowerShell</a>.</em>

**Comment by 237dmitry** [comment]
Broken console geometry on Linux [#21267](https://github.com/PowerShell/PowerShell/issues/21267)
